model/src/main/kotlin/config/PackageConfiguration.kt

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2017-2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.model.config
+
+import com.fasterxml.jackson.annotation.JsonInclude
+
+import com.here.ort.model.Identifier
+import com.here.ort.model.Provenance
+import com.here.ort.model.VcsType
+
+/**
+ * A configuration for a specific package and provenance. It allows to setup [PathExclude]s and (in the future, TODO)
+ * also license [LicenseFindingCuration]s, similar to how its done via the [RepositoryConfiguration] for projects.
```

Comment:
Remove "license".

Reply:
That sentence could also be part of the commit message.

Reply:
Done.